README.md

```diff
@@ -870,11 +870,11 @@ The **proper-arrows**/*this* rule can be configured in one of three modes: `"nes
 
 * [`"nested"`](#rule-this-configuration-nested) (default) permits a `this` to appear lower in a nested `=>` arrow function (i.e., `x = y => z => this.foo(z)`), as long as there is no non-arrow function boundary crossed.
 
-   Additionally, include the `"no-global"` option (default: `false`) to forbid `this`-containing `=>` arrow functions from the top-level scope (where they might inherit the global object as `this`).
+   Additionally, include the `"no-global"` option (default: `false`) to forbid `this`-containing `=>` arrow functions from the module-top-level scope (where they might inherit the global object as `this`).
```

Comment:
Actually, because modules are default strict mode, `this`-containing arrow functions in the top-level of a module have `this` as `undefined`, so I think this sentence should read:

> ...to forbid `this`-containing `=>` arrow functions from the global scope (where they might inherit the global object as `this`) or from the top-level of a module (where the `this` would be `undefined`).